pom.xml

```diff
@@ -313,6 +314,1094 @@
+        <gravitee-resource-oauth2-provider-generic.version>2.0.2</gravitee-resource-oauth2-provider-generic.version>
+        <gravitee-service-discovery-consul.version>1.3.0</gravitee-service-discovery-consul.version>
+        <!-- Management API Only -->
+        <gravitee-cockpit-connectors-ws.version>4.0.2</gravitee-cockpit-connectors-ws.version>
+        <gravitee-fetcher-bitbucket.version>2.0.1</gravitee-fetcher-bitbucket.version>
+        <gravitee-fetcher-git.version>2.0.0</gravitee-fetcher-git.version>
+        <gravitee-fetcher-github.version>2.0.1</gravitee-fetcher-github.version>
+        <gravitee-fetcher-gitlab.version>2.0.1</gravitee-fetcher-gitlab.version>
+        <gravitee-fetcher-http.version>2.0.1</gravitee-fetcher-http.version>
+        <gravitee-notifier-email.version>1.5.1</gravitee-notifier-email.version>
+        <gravitee-notifier-slack.version>1.3.0</gravitee-notifier-slack.version>
+        <gravitee-notifier-webhook.version>1.1.3</gravitee-notifier-webhook.version>
+        <!-- Gateway Only -->
+        <gravitee-reporter-elasticsearch.version>5.3.5</gravitee-reporter-elasticsearch.version>
+        <gravitee-reporter-file.version>3.2.1</gravitee-reporter-file.version>
+        <gravitee-reporter-tcp.version>2.3.1</gravitee-reporter-tcp.version>
+        <!--    Version of policy-ratelimit is also used for policy-quota, policy-spikearrest and gateway-services-ratelimit    -->
+        <!--    <gravitee-gateway-services-ratelimit.version>1.15.0</gravitee-gateway-services-ratelimit.version>    -->
+        <gravitee-tracer-jaeger.version>3.0.1</gravitee-tracer-jaeger.version>
+
+        <!-- Versions of the plugins for the full distribution on dev environment-->
+        <!-- Management API & Gateway -->
+        <!-- Community plugins -->
+        <gravitee-connector-kafka.version>1.1.1</gravitee-connector-kafka.version>
+        <gravitee-policy-aws-lambda.version>1.2.0</gravitee-policy-aws-lambda.version>
+        <gravitee-policy-circuit-breaker.version>1.1.5</gravitee-policy-circuit-breaker.version>
+        <gravitee-policy-geoip-filtering.version>2.0.0</gravitee-policy-geoip-filtering.version>
+        <gravitee-policy-javascript.version>1.3.3</gravitee-policy-javascript.version>
+        <gravitee-resource-auth-provider-http.version>1.3.2</gravitee-resource-auth-provider-http.version>
+        <gravitee-resource-auth-provider-inline.version>1.3.0</gravitee-resource-auth-provider-inline.version>
+        <gravitee-resource-auth-provider-ldap.version>1.3.0</gravitee-resource-auth-provider-ldap.version>
+        <gravitee-resource-cache-redis.version>1.5.0</gravitee-resource-cache-redis.version>
+        <gravitee-resource-oauth2-provider-keycloak.version>2.0.0</gravitee-resource-oauth2-provider-keycloak.version>
+        <gravitee-service-geoip.version>3.0.0</gravitee-service-geoip.version>
+        <gravitee-secretprovider-kubernetes.version>1.0.1</gravitee-secretprovider-kubernetes.version>
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Config Generic   
**Description:** Generic Config Credential   
**Severity:** HIGH   
**Match:** etprovider-kubernetes.version>*****</gravitee-secretpro  
[This comment was created by Aqua Pipeline]